tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -223,29 +223,11 @@ function Get-AzProfile
   return $ProfileMap
 }
 
-
-# Lists the profiles available for install from gallery
-function Get-ProfilesAvailable
-{
-  param([parameter(Mandatory = $true)] [PSCustomObject] $ProfileMap)
-  $ProfileList = ""
-  foreach ($Profile in $ProfileMap) 
-  {
-    foreach ($Module in ($Profile | get-member -MemberType NoteProperty).Name)
-    {
-      $ProfileList += "Profile: $Module`n"
-      $ProfileList += "----------------`n"
-      $ProfileList += ($Profile.$Module | Format-List | Out-String)
-    } 
-  }
-  return $ProfileList
-}
-
 # Lists the profiles that are installed on the machine
 function Get-ProfilesInstalled
 {
   param([parameter(Mandatory = $true)] [PSCustomObject] $ProfileMap, [REF]$IncompleteProfiles)
-  $result = @{}
+  $result = @{} 
```

Comment:
This should be OK, as long as this function is not exported and the calling function implements SHouldProcess